task_05.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Task 05"""
+
+
+def flip_keys(*to_flip):
+    """1.  A list named ``to_flip``. This list is assumed to have nested,
+        immutable sequences inside it, eg: ``[(1, 2, 3), 'hello']``"""
+    counter = 0
+    for flip in to_flip:
```

Comment:
inside your loop you'll want to alter the inner object. consider that your counter mimics the index of the object being looped (eg, when looping the first object the value of the counter is `0` and one the second item its value is `1` and so-on.). If you have that you can alter the main object `to_flip[counter] = flip[::-1]`